libs/libarchopenfpga/src/read_xml_bitstream_setting.cpp

```diff
@@ -102,6 +102,21 @@ static void read_xml_non_fabric_bitstream_setting(
   }
 }
 
+/********************************************************************
+ * Parse XML description for a bit setting under a <bit> XML node
+ *******************************************************************/
+static void read_xml_bit_setting(
+  pugi::xml_node& xml_bit, const pugiutil::loc_data& loc_data,
+  openfpga::BitstreamSetting& bitstream_setting) {
+  const std::string& path_attr =
+    get_attribute(xml_bit, "path", loc_data).as_string();
+  const std::string& value_attr =
+    get_attribute(xml_bit, "value", loc_data).as_string();
+  VTR_ASSERT_SAFE(value_attr == "0" || value_attr == "1");
```

Comment:
Force a strong check here. If an unexpect value is hit, error out.

Reply:
Changed to VTR_ASSERT(). Done via https://github.com/chungshien/OpenFPGA/commit/933155b08fe9e6b3d2d5bb3133ffbbfb0b5822dd

Reply:
ASSERT failures may confuse end users. Please use arch_throw() 

For example: https://github.com/lnis-uofu/OpenFPGA/blob/4a07a32902c1ce69ea083b9fe60b1910beb58b78/libs/libclkarchopenfpga/src/io/read_xml_clock_network.cpp#L55-L58

Reply:
Done via https://github.com/chungshien/OpenFPGA/commit/9882394c8bb2b7a05f168de9a8b080c22e665183